src/modules/servicenode.js

```diff
@@ -33,7 +46,9 @@ class ServiceNode {
             try {
                 // Read review record first. This caches the content, the history, and throws if it's not valid
                 this.chluIpfs.logger.debug('Reading and validating ReviewRecord ' + obj.multihash);
-                await this.chluIpfs.readReviewRecord(obj.multihash);
+                await this.chluIpfs.readReviewRecord(obj.multihash, {
+                    checkForUpdates: true
```

Comment:
@fazo96 I just want to be sure that we are checking for updates to RR in service nodes simply to make sure in case there was an update and we caught it. Or is there something more to it?

Reply:
when an update is made by the customer (or anyone) the WROTE_REVIEW_RECORD chlu event is emitted so it gets caught anyway, we don't need to check for updates all the time since it adds complexity.

`checkForUpdates` makes it so that when an update is found an event is emitted. This is very heavy since it creates a bunch of contexts and events listeners, when we make the custom store I'll rewrite it to be lighter.

now that we have the single db we also have `getLatestVersion` which returns the last update on the spot instead, and this one does not have the problems of the other.

I should improve documentation on this